src/js/angular/clustermanagement/directives.js

```diff
@@ -155,3 +274,26 @@ clusterManagementDirectives.directive('clusterGraphicalView', ['$window', 'Local
         };
     }
 ]);
+
+const getLegendNodes = function () {
+    const legendNodes = [];
+    legendNodes.push({nodeState: NodeState.LEADER, customText: 'Leader'});
```

Comment:
i18n

Reply:
I will add i18n for the whole cluster drawing in a following PR